### PR TITLE
IBX-6116: Scenario block added in 'Block reference' page

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://issues.ibexa.co/browse/IBX-4961)
| Versions      | master
| Edition       | Connect, Experience

Scenario block added in 'Block reference' page.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

**Review by Steveb-p — APPROVED** [review]

File: docs/content_management/block_reference.md [hunk 1]
```diff
@@ -141,6 +142,19 @@ On the **Basic** tab, set values in the following fields:
...
 Note that selecting a folder containing Content items other than images results in displaying only a link to the folder they are stored in.
 
+## Ibexa Connect block
+
+On the **Basic** tab, set values in the following fields:
+
+- **Name** – Enter a name for the Page block.
+- **Webhook link** - Enter the link for the Ibexa Connect webhook.
+
+On the **Design** tab, in the **View** field, select the template be used to present the webhook.
```
Comment (MagdalenaZuba) [inline docs/content_management/block_reference.md:152]:
```suggestion
In the **Design** tab, in the **View** field, select the template for the webhook.
```

File: docs/content_management/block_reference.md [hunk 2]
```diff
@@ -141,6 +142,19 @@ On the **Basic** tab, set values in the following fields:
 After submitting the settings, all images in the folder will appear in the Gallery block.
 Note that selecting a folder containing Content items other than images results in displaying only a link to the folder they are stored in.
 
+## Ibexa Connect block
+
+On the **Basic** tab, set values in the following fields:
```
Comment (MagdalenaZuba) [inline docs/content_management/block_reference.md:147]:
```suggestion
In the **Basic** tab, set values in the following fields:
```

**Review by MagdalenaZuba — APPROVED** [review]

File: docs/content_management/block_reference.md (hunk 2 above)
Comment (julitafalcondusza) [reply docs/content_management/block_reference.md:147]: Let's keep "on" for the page consistency.

File: docs/content_management/block_reference.md (hunk 1 above)
Comment (julitafalcondusza) [reply docs/content_management/block_reference.md:152]: Let's keep "on" for the page consistency.

**Pull request merged**